game/scripts/npc/abilities/sniper_keen_scope.txt

```diff
@@ -0,0 +1,15 @@
+"DOTAAbilities"
+{
+  "sniper_keen_scope"
+  {
+    // General
+    //-------------------------------------------------------------------------------------------------------------
+    "AbilityBehavior"                                     "DOTA_ABILITY_BEHAVIOR_PASSIVE | DOTA_ABILITY_BEHAVIOR_NOT_LEARNABLE"
+    "MaxLevel"                                            "6"
+
+    "AbilityValues"
+    {
+      "bonus_range"                                       "160 240 320 400 450 500"
```

Comment:
fuck yeah

Reply:
i keep thinking we should have a facet that replaces this talent with one that reduces base attack time :D :D :D smg sniper gogogo